src/viz-parole-success/VizParoleSuccess.js

```diff
@@ -0,0 +1,57 @@
+import { ascending } from "d3-array";
+import PropTypes from "prop-types";
+import React from "react";
+import MonthlyTimeseries from "../monthly-timeseries";
+import { addEmptyMonthsToData } from "../utils";
+
+function makeTimeseriesRecord(record) {
+  return {
+    month: `${record.year}-${record.month}`,
+    value: parseFloat(record.success_rate),
+  };
+}
+
+// consistently using `year` and `month` in all monthly data structures
+// makes it easier to generalize for common transformations
```

Comment:
I dig it, but should we handle key (re)naming upstream?  Probably more trouble than its worth.

Reply:
maybe? doing it here for now ... I kind of want to minimize the number of places that have deep knowledge of the shape of a particular data file. we can always generalize it more later